src/shared/containers/MapTest.js

```diff
@@ -5,30 +5,32 @@ import controllable from 'react-controllables';
 import GoogleMap from 'google-map-react';
 import Marker from '../components/Marker.js';
 
-class MapTest extends Component {  
-
-  
+class MapTest extends Component {
```

Comment:
What's the purpose of this test class? Please use enzyme for react component test. http://airbnb.io/enzyme/